app/src/main/java/at/roteskreuz/stopcorona/constants/Constants.kt

```diff
@@ -77,7 +77,6 @@ object Constants {
         const val DATA_PRIVACY_REPOSITORY_PREFIX = PREFIX + "terms_and_conditions_repository_"
         const val DASHBOARD_PREFIX = PREFIX + "dashboard_"
         const val QUARANTINE_REPOSITORY_PREFIX = PREFIX + "quarantine_repository_"
-        const val CORONA_DETECTION_REPOSITORY_PREFIX = PREFIX + "corona_detection_repository_"
```

Comment:
Note to self, are we clearing the value here?

Reply:
@falkorichter Already removed in https://github.com/austrianredcross/stopp-corona-android/pull/83/files#diff-4d7fe52c6e225b440d9b5efb6841d6a5R33